app/models/ZkKafka.scala

```diff
@@ -148,4 +160,14 @@ object ZkKafka {
 
     (new Totals(total, kafkaTotal, spoutTotal), deltas)
   }
+
+  // Attempt to parse a byte array as json, and catch and log any error rather than
+  // bubbling it up.
+  def tryParse(json: Array[Byte]): Option[JsValue] = try {
+    Some(Json.parse(json))
+  } catch {
+    case e: Exception =>
+      Logger.error(s"failed parsing json document:\n${json.take(100)}", e)
```

Comment:
Nit: Can you use `%n` instead of `\n` in Scala the way you can in Java? I assume so since my understanding of the `s"..."` syntax is that it's just syntactic sugar around `String.format`.